TFT/src/User/API/MachineParameters.c

```diff
@@ -218,3 +218,10 @@ void resetEepromSettings(void)
   if(infoMachineSettings.EEPROM == 1)
     mustStoreScript("M502\nM500\nM503 S0\n");
 }
+void saveEepromTouchmiSettings(void)
+{
+  if(infoMachineSettings.EEPROM == 1)
+    mustStoreCmd("M211 S1\n");
+     mustStoreCmd("G28 X Y\n");
+      mustStoreCmd("M500\n");
+}
```

Comment:
there is already a function for saving printer settings to eeprom EEPROM. See MachineParameters.c

Reply:
Yes ,i know, but after tuning the zoffset for TouchMi sensor, the hotend stay in the middle of printer.
I think it's better to bring the hotend back to home X and y, it's cleaner, isn't it ?